llvm-bazel/llvm-project-overlay/clang/BUILD

```diff
@@ -0,0 +1,1976 @@
+)
+
+gentbl(
+    name = "ast_type_properties_gen",
+    tbl_outs = [
+        (
+            "-gen-clang-type-reader",
+            "include/clang/AST/AbstractTypeReader.inc",
+        ),
+        (
+            "-gen-clang-type-writer",
+            "include/clang/AST/AbstractTypeWriter.inc",
+        ),
+    ],
+    tblgen = ":clang-tblgen",
+    td_file = "include/clang/AST/TypeProperties.td",
+    td_srcs = [
+        "include/clang/AST/PropertiesBase.td",
+        "include/clang/AST/TypeProperties.td",
+        "include/clang/Basic/ASTNode.td",
+        "include/clang/Basic/TypeNodes.td",
+    ],
+)
+
+gentbl(
+    name = "type_nodes_gen",
+    tbl_outs = [(
+        "-gen-clang-type-nodes",
+        "include/clang/AST/TypeNodes.inc",
+    )],
+    tblgen = ":clang-tblgen",
+    td_file = "include/clang/Basic/TypeNodes.td",
+    td_srcs = [
+        "include/clang/Basic/ASTNode.td",
+        "include/clang/Basic/TypeNodes.td",
+    ],
+)
+
+cc_library(
+    name = "ast",
+    srcs = glob([
+        "lib/AST/*.cpp",
+        "lib/AST/*.h",
+        "lib/AST/Interp/*.cpp",
+        "lib/AST/Interp/*.h",
+    ]),
+    hdrs = glob([
+        "include/clang/AST/*.h",
+    ]),
+    includes = [
+        "include",
+        "lib/AST/Interp",
+    ],
+    textual_hdrs = [
+        "include/clang/AST/AttrImpl.inc",
+        "include/clang/AST/AttrNodeTraverse.inc",
+        "include/clang/AST/Attrs.inc",
+        "include/clang/AST/AttrTextNodeDump.inc",
+        "include/clang/AST/AttrVisitor.inc",
+        "include/clang/AST/CommentCommandInfo.inc",
+        "include/clang/AST/CommentCommandList.inc",
+        "include/clang/AST/CommentHTMLNamedCharacterReferences.inc",
+        "include/clang/AST/CommentHTMLTags.inc",
+        "include/clang/AST/CommentHTMLTagsProperties.inc",
+        "include/clang/AST/CommentNodes.inc",
+        "include/clang/AST/DeclNodes.inc",
+        "include/clang/AST/StmtDataCollectors.inc",
+        "include/clang/AST/StmtNodes.inc",
+        "lib/AST/Interp/Opcodes.inc",
+    ] + glob([
+        "include/clang/AST/*.def",
+    ]),
+    deps = [
+        ":ast_attr_gen",
+        ":ast_comment_command_info_gen",
+        ":ast_comment_html_named_character_references_gen",
+        ":ast_comment_html_tags_gen",
+        ":ast_comment_html_tags_properties_gen",
+        ":ast_comment_nodes_gen",
+        ":ast_decl_nodes_gen",
+        ":ast_interp_opcodes_gen",
+        ":ast_properties_base_gen",
+        ":ast_stmt_data_collectors_gen",
+        ":ast_stmt_nodes_gen",
+        ":ast_type_properties_gen",
+        ":basic",
+        ":lex",
+        ":type_nodes_gen",
+        "//llvm:BinaryFormat",
+        "//llvm:Core",
+        "//llvm:FrontendOpenMP",
+        "//llvm:Support",
+    ],
+)
+
+cc_library(
+    name = "index",
+    srcs = glob(
+        [
+            "lib/Index/*.cpp",
+            "lib/Index/*.h",
+        ],
+        # Broken upstream after r367615, and currently unused.
```

Comment:
I can't find this revision

Reply:
Yeah, I've just removed the whole thing as it doesn't seem necessary any more.